ddtrace/llmobs/_evaluators/ragas/faithfulness.py

```diff
@@ -0,0 +1,22 @@
+import math
+import time
+
+
+class RagasFaithfulnessEvaluator:
+    LABEL = "ragas_faithfulness"
+    METRIC_TYPE = "score"
+
+    def __init__(self, llmobs_service):
+        self.llmobs_service = llmobs_service
+
+    def evaluate(self, span):
```

Comment:
Can we add typing here? Not sure what `span` is referring to exactly (dict? Span?)